samples/http_example/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "nodejs-docs-samples-iot-http-example",
+  "version": "0.0.1",
+  "description": "HTTP Example for Google Cloud IoT Core using NodeJS.",
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "main": "cloudiot_http_example_nodejs.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "scripts": {
+    "test": "repo-tools test install --cmd=npm -- run system-test",
+    "system-test": "mocha system-test/*.test.js --timeout=60000"
+  },
+  "dependencies": {
+    "jsonwebtoken": "^8.5.0",
+    "request": "^2.88.0",
+    "retry-request": "^4.0.0",
+    "yargs": "^15.0.0"
+  },
+  "devDependencies": {
+    "@google-cloud/nodejs-repo-tools": "^3.3.0",
```

Comment:
Remember how I said you could just drop these samples in, and they would work?  That was a bold faced lie.  In other places, we've stripped out the usage of `nodejs-repo-tools`.  Chances are we don't need it here, but we do want to make sure this gets removed.

Reply:
Some of the tests are flaky, this is a repo tools feature.

Reply:
repo-tools strategically removed now except for the tests that are async and race-condition-y, which are super touchy.